doc/src/build/sui-local-network.md

```diff
@@ -149,6 +149,46 @@ If successful, the response resembles the following:
 }
 ```
 
+## Connect Command Line Tool to Local Network
+
+```bash
+# If this is your first time creating a local network, create a new environment with alias `local` for local network
+sui client new-env --alias local --rpc http://127.0.0.1:9000
+
+# switch to the new local environment
+sui client switch --env local
+Active environment switched to [local]
+
+# confirm your local env
+sui client active-env
+local
+
+# show your sui address
+sui client active-address
+0xbc33e6e4818f9f2ef77d020b35c24be738213e64d9e58839ee7b4222029610de
+
+# request test tokens from local faucet
```

Comment:
This info is already in the section on using the local faucet

Reply:
Yes, I don't know which one to keep, feel free to make a decision and edit directly!